positron/moz.build

```diff
@@ -2,4 +2,93 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-DIRS += ['app', 'chrome', 'components', 'electron', 'modules', 'node', 'webidl']
+DIRS += ['app', 'chrome', 'components', 'modules', 'node', 'webidl']
+
+EXTRA_JS_MODULES.browser += [
```

Comment:
It's unfortunate that we have to list each module explicitly, especially because we'll have to touch this on Electron updates. But I suppose it has the advantage that we can choose to build only a subset of modules in the future.